dev/CommonStyles/AppBarButton_rs1_themeresources.xaml

```diff
@@ -1,5 +1,10 @@
 ﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
-<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
Should this file be renamed just to AppBarButton_themeresources.xaml, since it is no longer specific to RS1?

Reply:
I will keep the same name because it's easy to do the diff/blame. For default style of new controls, we may remove the rs1 but it's not the case.

---
In reply to: [259039176](https://github.com/Microsoft/microsoft-ui-xaml/pull/315#discussion_r259039176) [](ancestors = 259039176)

Reply:
I agree with keeping the names as-is for this review but we should take a follow-up to rename the files (with no changes) right after this.

Reply:
Actually thinking about this more, this review would be easier to look at if the changes were made to AppBarButton_rs5_themeresource.xaml instead so that we can see where conditionals get added or changes are made from the "latest" version of the control.

Reply:
disagree with this. rs5 mean it will be @RS5_ pages and be ignored in RS1 to RS4.

---
In reply to: [259966851](https://github.com/Microsoft/microsoft-ui-xaml/pull/315#discussion_r259966851) [](ancestors = 259966851)

Reply:
No, you can fix that by editing the vcxitems file:

```xml
    <Page Include="$(MSBuildThisFileDirectory)AppBarButton_rs5_themeresources.xaml">
      <Version>RS5</Version>
      <Type>ThemeResources</Type>
    </Page>
```

Change RS5 to RS1 in this change and then in a subsequent change rename AppBarButton_rs5_themeresources.xaml to AppBarButton_themeresources.xaml. That will give a more meaningful diff here.

Reply:
It will be a little heavy. I will need to create a new file to move rs5 themeresources like AppBarButtonSubItemChevronForegroundDisabled out to a new file since then are not the same between rs1 and rs5 

---
In reply to: [259971340](https://github.com/Microsoft/microsoft-ui-xaml/pull/315#discussion_r259971340) [](ancestors = 259971340)

Reply:
That was another one of my comments -- why are those separate in this change? Seems like we should just push the RS5 values down to RS1, except for reveal/acrylic, but those are pretty straightforward to do with conditional XAML, right?

Reply:
It's doable. some keys like SystemControlTransientBorderBrush, it doesn't exists in rs1. Something like AppBarButtonSubItemChevronForegroundPointerOver, we don't use it before RS4.

---
In reply to: [259973878](https://github.com/Microsoft/microsoft-ui-xaml/pull/315#discussion_r259973878) [](ancestors = 259973878)

Reply:
Keys not present on downlevel is something we can make better! That's a platform gap that's frustrating for control authors. Keys that we introduce but don't use -- is that because the platform control didn't support that functionality before RS4?

Reply:
yes. For example, SubItemChevron, visualstate change is trigged from code, so I excluded it on old version. Things like KeyboardAcceleratorText, it's existing in late API

---
In reply to: [259976740](https://github.com/Microsoft/microsoft-ui-xaml/pull/315#discussion_r259976740) [](ancestors = 259976740)